core/src/test/java/org/mskcc/oncotree/utils/CacheUtilTest.java

```diff
@@ -61,6 +60,12 @@
     
     @Resource(name="oncoTreeAdditionalVersionRepositoryMockResponse")
     private List<Version> oncoTreeAdditionalVersionRepositoryMockResponse;
+
+    @Resource(name="legacyVersion")
+    private Version legacyVersion;
+
+    @Resource(name="latestVersion")
```

Comment:
we will probably need additional versions (at least one more) to test the propagation/accumulation of history over time.